docs/public/make-scrollable-code-focusable.js

```diff
@@ -0,0 +1,3 @@
+Array.from(document.getElementsByTagName("pre")).forEach((element) => {
+  element.setAttribute("tabindex", "0");
```

Comment:
are `pre` elements tabbable? TIL

Reply:
They aren't by default unless you set `tabindex="0"` on them

Reply:
do we want them to be tabbable? I wouldn't think so, but assuming I'm missing a use-case?

Reply:
Yep we do! Since `<pre>` shouldn't wrap it overflows with a scrollbar. If the keyboard user can't select the `<pre>` via tab, they can't control the scrollbar [rule: Ensure that scrollable region has keyboard access](https://dequeuniversity.com/rules/axe/4.3/scrollable-region-focusable)

Reply:
ah, that makes sense! TIL, thanks for sharing that